tests/nightly/test_large_array.py

```diff
@@ -157,8 +157,8 @@ def test_take():
 
 
 def test_slice():
-    a = nd.ones(shape=(LARGE_X, SMALL_Y))
-    res = nd.slice(a, begin=(LARGE_X-1000, 1), end=(LARGE_X, SMALL_Y))
+    a = nd.ones(shape=(2, LARGE_SIZE))
```

Comment:
Why do we need to test 2 and update the LARGE_SIZE? I would suggest we keep this test file as it is and create a separate test file to test single dimension.